types/postcss-less/postcss-less-tests.ts

```diff
@@ -40,31 +51,40 @@ less.parse(lessCode).walkAtRules(atRule => {
     }
 });
 
-less.parse(lessCode).walkRules(rule => {
+less.parse('.a:extend(.b) {}').walkRules(rule => {
     const lessRule = rule as less.Rule;
     assert(lessRule.extend === true);
 });
 
-less.parse(lessCode).walkComments(comment => {
+less.parse('&:extend(.a)').walkDecls(decl => {
+    const lessDecl = decl as less.Declaration;
+    assert(lessDecl.extend === true);
+});
+
+less.parse('// a').walkComments(comment => {
     const lessComment = comment as less.Comment;
     assert(lessComment.inline === true);
 });
 
 less.stringify(less.parse(lessCode), (part, node, type) => {
     assert(typeof part === 'string');
-    assert(type === 'start');
-    assert(type === 'end');
```

Comment:
[note] The `type` argument is optional and this test code was incorrect. See below:

https://github.com/postcss/postcss/blob/7.0.32/lib/postcss.d.ts#L62